README.rst

```diff
@@ -37,4 +37,4 @@ Requirements
 ------------
 
 * **Python**: 2.7, 3.4+
-* **Backends**: In-memory (development), Postgresql 9.4+ (production)
+* **Backends**: In-memory (development), Postgresql 9.5+ (production)
```

Comment:
If we look [at the logs](https://gist.github.com/leplatrem/54637f78e6297ba8d5ba0b50d7447042), a part of the code already introspects it (see `select version()`).
I'll loook into SQLAlchemy...